package.json

```diff
@@ -61,6 +61,7 @@
     "@emotion/react": "^11.4.0",
     "@jupyter-widgets/base": "^1.1.10 || ^2.0.0 || ^3.0.0 || ^4.0.0",
     "@jupyterlab/application": "^3.0.11",
+    "@material-ui/core": "^4.12.3",
```

Comment:
Could we make the loader without adding material ui? Its a huge package to have in our bundle.